services/app-api/handlers/reports/update.ts

```diff
@@ -87,37 +84,23 @@ export const updateReport = handler(async (event, context) => {
 
 export const archiveReport = handler(async (event, context) => {
   let status, body;
-  const unvalidatedPayload = JSON.parse(event!.body!);
-
   // get current report
   const reportEvent = { ...event, body: "" };
   const getCurrentReport = await fetchReport(reportEvent, context);
 
   if (getCurrentReport?.body) {
-    if (unvalidatedPayload?.archived) {
-      const validatedArchiveValue = await validateData(
-        archiveValidationSchema,
-        unvalidatedPayload
-      );
-      const currentReport = JSON.parse(getCurrentReport.body);
-      const reportParams = {
-        TableName: process.env.MCPAR_REPORT_TABLE_NAME!,
-        Item: {
-          ...currentReport,
-          lastAltered: Date.now(),
```

Comment:
per discussion with David Koger we are omitting `lastAltered` and `lastAlteredBy` from the archive-type update because it changes nothing about the report and we don't want an admin showing up as an editing user